lib/flipper/engine.rb

```diff
@@ -25,6 +25,7 @@ def self.default_strict_value
         log: ENV.fetch('FLIPPER_LOG', 'true').casecmp('true').zero?,
         cloud_path: "_flipper",
         strict: default_strict_value,
+        actor_limit: ENV["FLIPPER_ACTOR_LIMIT"]&.to_i || 100,
```

Comment:
100% thought this would return 0 but tested in ira and it was 100. TIL.

Reply:
Pretty sure this was a Copilot suggestion